src/Bundle/Controller/Parameters.php

```diff
@@ -14,23 +14,61 @@
 namespace Sylius\Bundle\ResourceBundle\Controller;
 
 use Symfony\Component\HttpFoundation\ParameterBag;
+use Symfony\Component\HttpKernel\Kernel;
 
-class Parameters extends ParameterBag
-{
-    /**
-     * @param string $key
-     * @param mixed $default
-     *
-     * @return mixed
-     */
-    public function get($key, $default = null)
+if (Kernel::MAJOR_VERSION === 6) {
```

Comment:
I believe we can skip check for Symfony 6 for now, fix the build with Symfony 4.4 and 5.4 **and then** drop 4.4 and introduce Symfony 6 properly, wdyt?

Reply:
You're right... it's done.